app/services/staff_action_logger.rb

```diff
@@ -990,6 +990,14 @@ def log_destroy_public_sidebar_section(section)
     )
   end
 
+  def log_reset_bounce_score(user, opts = {})
```

Comment:
I didn't unit test this because it's integration tested in `spec/requests/admin/users_controller_spec.rb`